test/tbb/test_limiter_node.cpp

```diff
@@ -419,6 +419,21 @@ void test_decrementer() {
         CHECK_MESSAGE( actual == expected2[m++], "" );
     CHECK_MESSAGE( ( sizeof(expected2) / sizeof(expected2[0]) == m), "Not all messages have been processed." );
     g.wait_for_all();
+
+    const size_t threshold3 = 10;
+    tbb::flow::limiter_node<int, long long> limit3(g, threshold3);
+    make_edge(limit3, queue);
+    long long decrement_value3 = 1;
```

Comment:
Let's make it equal to some non-obvious value? For example,
```suggestion
    long long decrement_value3 = 3;
```